### ISSUE TITLE
memo no longer in startup event?

### ISSUE THREAD

**Issue by masantiago** [nolar/kopf#700]
## Question

By doing a basic example with kopf:

```python
@kopf.on.startup()
async def startup(memo: kopf.Memo, **_):
    pass

TypeError: startup() missing 1 required positional argument: 'memo'
```

Nevertheless, it works with other handlers:

```python
@kopf.on.create("example.com")
async def create(spec, memo: kopf.Memo, **kwargs):
```

I am using the latest version of kopf 1.29.2

## Checklist

- [ X] I have read the [documentation](https://kopf.readthedocs.io/en/latest/) and searched there for the problem
- [ X] I have searched in the [GitHub Issues](https://github.com/nolar/kopf/issues?utf8=%E2%9C%93&q=) for similar questions

## Keywords

memo

**Comment by nolar** [comment]
Hello 

The global memos are only available in `kopf==1.30.0rc1` — this is a pre-release yet, so it will not be installed without the version explicitly specified or without the “pre” flag: `pip install --pre kopf`.

* https://github.com/nolar/kopf/releases/tag/1.30.0rc1

Rel: #667

**Comment by masantiago** [comment]
Thank you @nolar! I was confused by reading the doc. Now it works perfect!
